ocl/OCL.g4

```diff
@@ -0,0 +1,326 @@
+/******************************
+* Parser for OCL 2.4 with extensions for textual notations
+* for UML classes and usecases. 
+*
+* Arrow operators ->op are used consistently for any OCL 
+* operator, not just collection operators. 
+* 
+* Copyright (c) 2003--2022 Kevin Lano
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0
+*
+* SPDX-License-Identifier: EPL-2.0
+* *****************************/
+
+grammar OCL;	
+	
+specification:
+  'package' ID '{' classifier* '}' EOF;
+
+classifier: classDefinition
+    | interfaceDefinition
+    | usecaseDefinition
+    | enumeration;
+
+interfaceDefinition:	 'interface' ID '{' classBody? '}' 
+    | 'interface' ID 'extends' ID '{' classBody? '}'; 
```

Comment:
The common part can be extracted here and below:

```suggestion
interfaceDefinition
    : 'interface' ID ('extends' ID)? '{' classBody? '}'
    ;
```

Also, please use such formatting (we use it across all grammars in the repository):

```
rule
    : a1 (a2 | a3)
    | a4
    ...
    | an
    ;
```